README.md

```diff
@@ -1 +1,54 @@
-# Roman-Numeral-Converter
\ No newline at end of file
+readme# Project Title 🚀
+Roman-Numeral-Converter
+## Project Description 📝
+
+>HTML contains an input element with an id of "number" a button with an id of "convert-btn" and a paragraph with an id of "output".
+
+on CSS styles I moved the page to center the content and add basic styles
+
+The JavaScript includes:
+
+An event listener for the button click to handle the conversion process.
+Input validation to ensure the number is within the valid range (1 to 3999).
+A function convertToRoman that converts a number to its Roman numeral equivalent.
+
+
+## Demo 📸
+![Demo]
+(Roman-NC.png)
+
+## Technologies Used 🛠️
+
+- HTML
+- CSS
+-JS
+
+## Installation 💻
+
+Provide step-by-step instructions on how to install and set up your project. Include any dependencies or prerequisites that need to be installed.
+
+```bash
+# git@github.com:latifa-wakili/Roman-Numeral-Converter.git.
+```
+
+```bash
+# cd Roman-Numeral-Converter.
+```
+
+# vscode.
+```
```

Comment:
- Remove this and paste this instead:
  - Dont forget to remove the parenthesis after you pasted `( )`

```
(```
code .
```)
```